### PR TITLE
Updating Last (lastal) to v874

### PR DESCRIPTION
**Pull request by jmeppley**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Adds the 8 byte versions of of the binaries (lastdb8, lastal8, ...) to the build for indexing and searching HUGE dbs on high memory machines.

### CODE REVIEW

File: recipes/last/build.sh [hunk 1]
```diff
@@ -20,12 +23,14 @@ last-map-probs \
 last-dotplot \
 "
 
-export CMAKE_CXX_FLAGS="-std=c++11 -stdlib=libc++"
+if [[ $SUBDIR == osx* ]]; then
```
Comment (daler) [inline recipes/last/build.sh]: @jmeppley how does `$SUBDIR` get set?

**Comment by jmeppley** [comment]
Yeah, that's a bit of a hack. It's part of the conda build process.

I was just trying to get it to compile at the time, and meant to go back to
the conda docs to look for a better way to get the os.

I'll look one more time before merging...

**Comment by daler** [comment]
I didn't see `$SUBDIR` in https://conda.io/docs/user-guide/tasks/build-packages/environment-variables.html which is why I was asking, but this should do the trick without having to worry about env vars:

```bash
if [[ $(uname) == "Darwin" ]]; then
  export CMAKE_CXX_FLAGS="-stdlib=libc++"
fi
```

**Comment by jmeppley** [comment]
I'll update the PR to use that instead. Thanks!

**Comment by tomkinsc** [comment]
Thanks for this, lastal is normally an annoying package to update because of the 2to3 patching. If you've been using the updated version, have you encountered any issues running it under py3? Especially since it looks like the patch mostly shifts line numbers down relative to a prior patch.

**Comment by jmeppley** [comment]
I honestly have never used the maf-convert tool (the one that needs the patch), so I haven't tested it beyond checking if it compiles.

I don't even use last's MAF format, so I don't have anything lying around to test it on. Have you seen problems with the latest version?

**Comment by jmeppley** [comment]
I was able to test the converted python script on osx, and it seems fine to me.

**Comment by tomkinsc** [comment]
No problems so far! Thanks for the update!

**Pull request merged**